src/metabase/embedding/settings.clj

```diff
@@ -147,32 +149,28 @@
        (str/join " ")
        str/trim))
 
-(mu/defn- add-localhost :- :string [s :- [:maybe :string]]
-  (->> s ignore-localhost (str "localhost:* ") str/trim))
-
 (defn- -embedding-app-origins-sdk []
-  (add-localhost (setting/get-value-of-type :string :embedding-app-origins-sdk)))
+  (setting/get-value-of-type :string :embedding-app-origins-sdk))
 
 (defn- -embedding-app-origins-sdk!
   "The setter for [[embedding-app-origins-sdk]].
 
   Checks that we have SDK embedding feature and that it's enabled, then sets the value accordingly."
   [new-value]
-  (add-localhost ;; return the same value that is returned from the getter
-   (->> new-value
-        ignore-localhost
-        ;; Why ignore-localhost?, because localhost:* will always be allowed, so we don't need to store it, if we
-        ;; were to store it, and the value was set N times, it would have localhost:* prefixed N times. Also, we
-        ;; should not store localhost:port, since it's covered by localhost:* (which is the minumum value).
-        (setting/set-value-of-type! :string :embedding-app-origins-sdk))))
+  (->> new-value
```

Comment:
Removes the `add-localhost`. We do not need to add `localhost:*` to the embedding app origins, as it is always allowed in the middleware regardless of whether the SDK is enabled or not.